### PR TITLE
Update people.json

### PR DESCRIPTION
**Pull request by artofmel**
Added Mel Cummings

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -5690,4 +5690,27 @@
          ]
       ]
    }
```
Comment (annarankin) [inline people.json]: Looks like we're missing a comma here - that'll cause errors when the script tries to parse this file:
```suggestion
   },
```

File: people.json [hunk 2]
```diff
@@ -5690,4 +5690,27 @@
...
+               "Twitter",
+               "https://twitter.com/artofmelc"
+            ],
+            [
+               "Instagram",
+               "https://instagram.com/artofmelc"
+            ],
+      ]
+   }
 ]
```
Comment (annarankin) [inline people.json]: One more sneaky syntax error in the form of a missing curly bracket:
```suggestion
               "https://instagram.com/artofmelc"
            ]
         ]
      }
   }
]
```

**Review by annarankin — CHANGES_REQUESTED** [review]
Just two small syntax issues; once those are fixed this is good to go!
image

File: people.json (hunk 2 above)
Comment (artofmel) [reply people.json]: Hi Anna, Thanks for checking this over. I think I have made the requested changes now in the updated commit (759a385). Do let me know if this still needs addressing, thanks.

**Review by annarankin — APPROVED** [review]
Good to go! 🙌

**Pull request merged**